snapcraft/internal/sources/errors.py

```diff
@@ -21,6 +21,22 @@ class VCSError(errors.SnapcraftError):
     fmt = '{message}'
 
 
+class UnhandledSourceError(errors.SnapcraftError):
+
+    fmt = ('no handler to manage source ({source})')
```

Comment:
Can you use the format we agreed on which @elopio has worked on a lot?

Reply:
It took me a while to think of proper wording here... hope you like what I put in there now.